configuration/hardening.md

```diff
@@ -56,6 +56,10 @@ Gravwell uses the [bcrypt](https://en.wikipedia.org/wiki/Bcrypt) hashing system
 
 We start with a pretty aggressive bcrypt hash cost of 12 and routinely re-evaluate whether we need to increase that cost. 
 
+## Enforcing MFA
+
+[Requiring MFA](/configuration/mfa/mfa) will force all Gravwell users (except [SSO](/configuration/sso) users, who authenticate separately) to configure a second authentication method for login, e.g. an authenticator app. This makes brute-forcing logins essentially impossible and is highly recommended if the Gravwell server is exposed to the Internet.
```

Comment:
Nit: Missing comma after "e.g."--`... for login, e.g., an authenticator...`

Reply:
Brief research indicates this is largely a matter of style and since it is followed by just one example rather than a list, I'm going to keep it as is. I have a tendency to overuse commas in general.

Reply:
Very agreed that grammar is just a set of opinionated "rules" \o/